apps/fluent-tester/src/TestComponents/VibrancyView/VibrancyViewTest.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react';
+import { Alert, StyleSheet, View } from 'react-native';
+import type { ViewProps } from 'react-native';
+
+import { ButtonV1 as Button, Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import type { Material, BlendingMode, State } from '@fluentui-react-native/vibrancy-view';
+import { VibrancyView } from '@fluentui-react-native/vibrancy-view';
+
+import { VIBRANCYVIEW_TESTPAGE } from '../../../../E2E/src/VibrancyView/consts';
+import { MenuPicker } from '../Common/MenuPicker';
+import { stackStyle, commonTestStyles as commonStyles } from '../Common/styles';
+import type { TestSection, PlatformStatus } from '../Test';
+import { Test } from '../Test';
+
+const styles = StyleSheet.create({
+  box: {
+    width: 100,
+    height: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  bigBox: {
+    width: 200,
+    height: 200,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  biggerBox: {
+    backgroundColor: 'yellow',
+    width: 300,
+    height: 300,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  Rectangle1: {
+    backgroundColor: 'yellow',
+    width: 300,
+    height: 300,
+  },
+  Rectangle2: {
+    position: 'absolute',
+    top: 50,
+    width: 400,
+    height: 200,
+  },
+  centeredBox: {
+    backgroundColor: 'green',
+    width: 100,
+    height: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  borderProps: {
+    backgroundColor: 'blue',
+    borderColor: 'red',
+    borderWidth: 1,
+    borderRadius: 8,
+  },
+  rootWithSpacing: { ...commonStyles.root, gap: 8 },
+});
+
+const VibrancyViewDefault: React.FunctionComponent = () => {
+  return (
+    <Stack style={stackStyle}>
+      <View style={styles.rootWithSpacing}>
+        <Text>Vibrancy View with no props set</Text>
+        <VibrancyView style={styles.box} />
+      </View>
+    </Stack>
+  );
+};
+
+const VibrancyViewWithViewProps: React.FunctionComponent = () => {
+  const descriptionString = '<View> and <VibrancyView> with the same props set';
```

Comment:
Might make it clearer what we're looking at if you specified what props are being set?